stdlib/camlinternalFormat.ml

```diff
@@ -537,6 +544,16 @@ let bprint_fmt buf fmt =
     | Theta rest ->
       buffer_add_char buf '%'; bprint_ignored_flag buf ign_flag;
       buffer_add_char buf 't'; fmtiter rest false;
+    | Custom (arity, _, rest) ->
+      let n = int_of_custom_arity arity in
+      if n > 0 then begin
+        buffer_add_char buf '%'; bprint_ignored_flag buf ign_flag;
+        buffer_add_char buf '?';
+        for _i = 1 to n - 1 do
+          buffer_add_string buf "%?"
```

Comment:
I wasn't sure about this part, I think `ign_flag=true` is actually impossible here. But for uniformity with the other cases it make sense to use `bprint_ignored_flag` everywhere indeed. I'll update the patch.